### PR TITLE
Fix compatibility with PVR Addon API 5.5.0

### PR DESCRIPTION
**Pull request by fetzerch**
Minimalistic changes to make the addon compatible again with the current kodi v18 master.

Related PRs: https://github.com/xbmc/xbmc/pull/12552, https://github.com/xbmc/xbmc/pull/12598, https://github.com/xbmc/xbmc/pull/12609

### CODE REVIEW

File: pvr.mythtv/changelog.txt [hunk 1]
```diff
@@ -1,3 +1,6 @@
+v5.2.0
```
Comment (ksooo) [inline pvr.mythtv/changelog.txt:1]: changelog version number does not match version number in addon.xml.in (5.1.4 vs 5.2.0)

File: src/pvrclient-mythtv.cpp [hunk 2]
```diff
@@ -942,7 +941,6 @@ PVR_ERROR PVRClientMythTV::GetRecordings(ADDON_HANDLE handle)
       tag.iLifetime = 0;
       tag.iPriority = 0;
       PVR_STRCPY(tag.strPlotOutline, "");
-      PVR_STRCPY(tag.strStreamURL, "");
```
Comment (ksooo) [inline src/pvrclient-mythtv.cpp]: No need to delete this, because PVR_RECORDING::strStreamURL was not removed.

File: src/pvrclient-mythtv.cpp [hunk 3]
```diff
@@ -1052,7 +1050,6 @@ PVR_ERROR PVRClientMythTV::GetDeletedRecordings(ADDON_HANDLE handle)
...
```
Comment (ksooo) [inline src/pvrclient-mythtv.cpp]: No need to delete this, because PVR_RECORDING::strStreamURL was not removed.

**Comment by fetzerch** [comment]
Thanks @ksooo, fixed.

**Comment by ksooo** [comment]
Have you runtime-tested the changes? Does channel playback work?

**Comment by fetzerch** [comment]
Yes, sure. Playing a channel works, and switching works as well.

**Comment by ksooo** [comment]
Cool. Thanks. Up to @janbar to merge.

**Comment by janbar** [comment]
Thanks Christian.

**Pull request merged**